RELEASE-LEADS.md

```diff
@@ -238,61 +154,92 @@ green light.
 
-### Pin `serving` and `eventing` releases in dependent repositories
+- [knative/eventing](https://github.com/knative/eventing)
+- [knative-sandbox/discovery](https://github.com/knative-sandbox/discovery)
 
-**After** the tags for `serving` and `eventing` are created, their version needs
-to be pinned in all repositories that depend on them.
+- [knative-sandbox/sample-controller](https://github.com/knative-sandbox/sample-controller)
 
-For **serving** that is:
+After **serving**:
 
-- [knative/client](https://github.com/knative/client)
 - [knative-sandbox/net-kourier](https://github.com/knative-sandbox/net-kourier)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib)
 
-For **eventing** that is:
+After **eventing**:
 
-- [knative/client](https://github.com/knative/client)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib)
+- [knative-sandbox/eventing-awssqs](https://github.com/knative-sandbox/eventing-awssqs)
+- [knative-sandbox/eventing-camel](https://github.com/knative-sandbox/eventing-camel)
+- [knative-sandbox/eventing-ceph](https://github.com/knative-sandbox/eventing-ceph)
+- [knative-sandbox/eventing-couchdb](https://github.com/knative-sandbox/eventing-couchdb)
+- [knative-sandbox/eventing-kafka](https://github.com/knative-sandbox/eventing-kafka)
 - [knative-sandbox/eventing-kafka-broker](https://github.com/knative-sandbox/eventing-kafka-broker)
+- [knative-sandbox/eventing-natss](https://github.com/knative-sandbox/eventing-natss)
+- [knative-sandbox/eventing-prometheus](https://github.com/knative-sandbox/eventing-prometheus)
+- [knative-sandbox/eventing-rabbitmq](https://github.com/knative-sandbox/eventing-rabbitmq)
+- [knative-sandbox/sample-source](https://github.com/knative-sandbox/sample-source)
 
-The pins are similar to step 5 above, but now we're pinning `serving` and
-`eventing` respectively. Again, the pin PRs are sent against the **master**
-branch of each repository respectively.
+After both **eventing** and **serving**:
 
-### Cut `release-x.y` branches of all remaining repositories
+- [knative-sandbox/eventing-redis](https://github.com/knative-sandbox/eventing-redis)
+- [knative-sandbox/eventing-github](https://github.com/knative-sandbox/eventing-github)
+- [knative-sandbox/eventing-gitlab](https://github.com/knative-sandbox/eventing-gitlab)
 
-After the pin PRs are merged, cut the `release-x.y` branch in each of the
-remaining repositories (except `operator` and `client` as they are cut
-separately by the respective working group):
+Lastly:
 
-- [knative-sandbox/net-kourier](https://github.com/knative-sandbox/net-kourier)
-- [knative/eventing-contrib](https://github.com/knative/eventing-contrib)
-- [knative-sandbox/eventing-kafka-broker](https://github.com/knative-sandbox/eventing-kafka-broker)
+- [knative-sandbox/eventing-autoscaler-keda](https://github.com/knative-sandbox/eventing-autoscaler-keda)
+
+We have a few repos inside of Knative that are not handled in the standard
```

Comment:
Could you explain a bit more on this? And do we have plans to get these repos to be handled by the standard process as well?

Reply:
It is up to those working groups. They do not leverage the release leads process yet.